nixos/modules/hardware/video/nvidia.nix

```
@@ -139,9 +142,54 @@ in
       description = ''
         Enable render offload support using the NVIDIA proprietary driver via PRIME.
 
-        If this is enabled, then the bus IDs of the NVIDIA and Intel GPUs have to be
-        specified (<option>hardware.nvidia.prime.nvidiaBusId</option> and
-        <option>hardware.nvidia.prime.intelBusId</option>).
+        If this is enabled, then the bus IDs of the NVIDIA and Intel/AMD GPUs have to
+        be specified (<option>hardware.nvidia.prime.nvidiaBusId</option> and
+        <option>hardware.nvidia.prime.intelBusId</option> or
+        <option>hardware.nvidia.prime.amdgpuBusId</option>).
+      '';
+    };
+
+    hardware.nvidia.prime.offload.enableOffloadCmd = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Adds a `nvidia-offload` convenience script to <option>environment.systemPackages</option>
+        for offloading programs to an nvidia device. To work, should have also enabled
+        <option>hardware.nvidia.prime.offload.enable</option> or <option>hardware.nvidia.prime.reverse_sync.enable</option>
+
+        Example usage `nvidia-offload sauerbraten_client`
+      '';
+    };
+
+    hardware.nvidia.prime.reverse_sync.enable = mkOption {
```

We generally prefer `camelCase` instead of `snake_case` for options:

```suggestion
    hardware.nvidia.prime.reverseSync.enable = mkOption {
```